testapps/TestLayerExample/Readme.md

```diff
@@ -0,0 +1,49 @@
+# AWS Lambda Empty Function Project
+
+This starter project consists of:
+* Function.cs - class file containing a class with a single function handler method
+* aws-lambda-tools-defaults.json - default argument settings for use with Visual Studio and command line deployment tools for AWS
+
+You may also have a test project depending on the options selected.
+
+The generated function handler is a simple method accepting a string argument that returns the uppercase equivalent of the input string. Replace the body of this method, and parameters, to suit your needs. 
+
+## Here are some steps to follow from Visual Studio:
```

Comment:
Does everything work from Visual Studio?  
I remember reading a note earlier in the PR that some things won't work.

Reply:
Once Visual Studio is updated with this code it will be able to deploy-functions that are referencing layers.

This README.md file is for a testapp uses for the integration tests. Not a deliverable artifact.